### PR TITLE
koordlet: disable unnecessary collector and informer is blkio qos fea…

### PR DESCRIPTION
**Pull request by zwzhang0107**
…ture disabled

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: docker/koordlet.dockerfile [hunk 1]
```diff
@@ -19,6 +19,7 @@ RUN GOOS=linux GOARCH=amd64 go build -a -o koordlet cmd/koordlet/main.go
 # https://gitlab.com/nvidia/container-images/cuda/-/tree/master/dist.
 
 FROM nvidia/cuda:11.2.2-base-ubuntu20.04
+RUN apt-get update && apt-get upgrade -y lvm2 && rm -rf /var/cache/apt/
```
Comment (jasonliu747) [inline docker/koordlet.dockerfile:22]: dockerfile这个改动，我看已经另外提交一个PR了

Comment (zwzhang0107) [reply docker/koordlet.dockerfile]: yes

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**